data/installed-tests/fwupdmgr.sh

```diff
@@ -43,7 +43,7 @@ rc=$?; if [[ $rc != 0 ]]; then error $rc; fi
 
 # ---
 echo "Installing test firmware..."
-fwupdmgr install ${dirname}/fakedevice124.cab
+fwupdmgr update $device -y
```

Comment:
is this testing the same thing? where does the metadata come from?

Reply:
It's pulling from local `fwupd-tests` metadata.  It should be same file.  It let me run it in tree without having to manually fixup paths.

Reply:
Okay, sounds good then. I guess we're testing the checksum verification rather than installing a naked file, although it does seems to shame to loose the local-file-testing too.